js/models/contact_model.js

```diff
@@ -197,7 +197,7 @@ angular.module('contactsApp')
 					}
 				} else {
 					// getter
-					var property = this.validate('categories', this.getProperty('categories'));
+					var property = this.getProperty('categories');
```

Comment:
These are unrelated changes, right? Or do they have anything to do with issue #129 ?

Reply:
The validate is now integrated into the getProperty function :)